.env.example

```
@@ -1,2 +1,12 @@
\ No newline at end of file
+POSTGRES_USER=postgres
+
+# Trust Missing Avatars Helper
+# DATABASE_URL=postgres://postgres:postgres@localhost:5432/circles
+# RPC_URL=https://rpc.chiadochain.net
+# PRIVATE_KEY=0x...
+# ROUTER_ADDRESS=0x...
+# HUB_ADDRESS=0xc12C1E50ABB450d6205Ea2C3Fa861b3B834d13e8
+# BATCH_SIZE=50
+# CHECK_HUMANS=true
+# HUMAN_CHECK_CONCURRENCY=25
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)